src/natcap/invest/spec_utils.py

```diff
@@ -93,15 +93,15 @@
     "name": gettext("precipitation")
 }
 ET0 = {
-    "name": gettext("evapotranspiration"),
+    "name": gettext("reference evapotranspiration"),
     "type": "raster",
     "bands": {
         1: {
             "type": "number",
             "units": u.millimeter
         }
     },
-    "about": gettext("Map of evapotranspiration values.")
+    "about": gettext("Map of reference evapotranspiration values.")
```

Comment:
I confirmed that the 3 models that use `spec_utils.ET0` all expect "reference" evapotranspiration. They are SWY, Urban Cooling, and AWY.